packages/dds/tree/src/test/feature-libraries/modular-schema/modularChangeFamily.spec.ts

```diff
@@ -0,0 +1,229 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { JsonCompatibleReadOnly } from "../../../change-family";
+import {
+    FieldChangeEncoder,
+    FieldChangeHandler,
+    FieldChangeMap,
+    FieldChangeRebaser,
+    FieldKind,
+    Multiplicity,
+    ModularChangeFamily,
+    FieldKinds,
+} from "../../../feature-libraries";
+import { FieldKindIdentifier } from "../../../schema-stored";
+import { Delta, FieldKey } from "../../../tree";
+import { brand } from "../../../util";
+
+type ValueChangeset = FieldKinds.ReplaceOp<number>;
+
+const valueHandler: FieldChangeHandler<ValueChangeset> = {
+    rebaser: FieldKinds.replaceRebaser(),
+    encoder: new FieldKinds.ValueEncoder<ValueChangeset & JsonCompatibleReadOnly>(),
+
+    intoDelta: (change, deltaFromChild) => [{
+        type: Delta.MarkType.Modify,
+        setValue: (change as FieldKinds.Replacement<number>).new,
+    }],
```

Comment:
It looks like you may need to check wether `change` is `0` and return `[]` in that case.
I'm guessing that's not needed for the tests now, but the tests are likely going to be augmented with some fuzzing.
That should get rid of the cast.

Reply:
Done